dmoj/graders/communication.py

```diff
@@ -73,6 +74,13 @@ def populate_result(self, error, result, process):
             result.result_flag |= Result.TLE
 
     def check_result(self, case, result):
+        checker_name = case.config['checker'] or 'standard'
+        if isinstance(checker_name, ConfigNode):
+            checker_name = checker_name['name']
+
+        if checker_name != 'standard':
```

Comment:
can be sort to (`case.config['checker'] or 'standard') != 'standard'), since we do not care much about the name later.